pypopquiz/video.py

```diff
@@ -12,18 +12,11 @@
 VideoBackend = ppq.backends.backend.Backend
 
 
-def get_interval_in_s(interval: List[str]) -> List[int]:
-    """Converts an interval in string form (e.g. [1:10, 2:30] in seconds, e.g. [70, 150] seconds"""
-    return [int(sec.split(":")[0]) * 60 + int(sec.split(":")[1]) for sec in interval]
-
-
-def filter_stream(stream: VideoBackend, kind: str, round_id: int, question: Dict, question_id: int,
-                  box_height: int = 100, fade_amount_s: int = 3,
+def filter_stream(stream: VideoBackend, kind: str, round_id: int, answer_text: str, question_id: int,
```

Comment:
These function calls (`filter_stream` and `create_video`) feel a but clunky (many arguments, controlling things at different levels of abstraction). I don't have a direct solution for this: I suspect that grouping create_video and filter_stream in a class might be able to clean things up, since they need access to many of the the same data elements from our input file, and can share a set of common default configuration variables (like the output width / height, duration / properties of effects). Something to think about for the future.